bugsnag-android-core/src/main/java/com/bugsnag/android/SessionStore.kt

```diff
@@ -13,14 +12,14 @@ import java.util.Date
  * lack of network connectivity.
  */
 internal class SessionStore(
-    private val config: ImmutableConfig,
+    bugsnagDir: File,
+    maxPersistedSessions: Int,
+    private val apiKey: String,
     logger: Logger,
     delegate: Delegate?
 ) : FileStore(
-    File(
-        config.persistenceDirectory.value, "bugsnag/sessions"
-    ),
-    config.maxPersistedSessions,
+    File(bugsnagDir, "sessions"),
```

Comment:
Are we consistent in when we add a `bugsnag` subdir under the persistence directory?

Reply:
I see now - these are the uses where we go through `migrateLegacyFiles`; the others are using the config value directly.